script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -1196,12 +1213,12 @@ She suppresses her emotions from violently breaking out from within in some way.
 [sha:a29f2e4edf03ae504d9a740d0dd53948e27f98e4]{
 -- Page 385, Offset 25242.
 -- 名前が知れただけで充分だ。あとは直接見つけて、この目で確かめればいいさ」
-Knowing his name is more than enough. Now all we have to do is find him in person and confirm it with these eyes."
+Knowing his name is more than enough. Now all we have to do is find him in person and confirm it with my eyes."
```

Comment:
my own eyes? same thing but i just like it imo

Reply:
that reads pretty stilted imo